x-pack/plugins/ml/public/application/overview/components/analytics_panel/analytics_panel.tsx

```diff
@@ -145,27 +175,25 @@ export const AnalyticsPanel: FC<Props> = ({
                 </h3>
               </EuiText>
             </EuiFlexItem>
-            {analyticsStats !== undefined && (
-              <EuiFlexItem grow={false} className="mlOverviewPanel__statsBar">
-                <StatsBar stats={analyticsStats} dataTestSub={'mlOverviewAnalyticsStatsBar'} />
-              </EuiFlexItem>
-            )}
+            <EuiFlexItem grow={false}>
+              <EuiFlexGroup gutterSize={'s'} alignItems="center">
+                {analyticsStats !== undefined ? (
+                  <EuiFlexItem grow={false}>
+                    <StatsBar stats={analyticsStats} dataTestSub={'mlOverviewAnalyticsStatsBar'} />
+                  </EuiFlexItem>
+                ) : null}
+                <EuiFlexItem grow={false}>
+                  <EuiButton size="m" fill href={manageJobsLink}>
+                    {i18n.translate('xpack.ml.overview.analyticsList.manageJobsButtonText', {
+                      defaultMessage: 'Manage jobs',
+                    })}
+                  </EuiButton>
+                </EuiFlexItem>
+              </EuiFlexGroup>
+            </EuiFlexItem>
           </EuiFlexGroup>
           <EuiSpacer />
           <AnalyticsTable items={analytics} />
```

Comment:
Would be useful to have a way of linking to the DFA jobs list, filtered to show just the selected job. Either an extra 'Manage job' action, or the tables in Stack Management use a link from the job ID.

Reply:
Added in b2dcb531e2b849a57fd5a0a204b0027cf79a7d10